test/e2e/case15_event_format_test.go

```diff
@@ -62,11 +63,23 @@ var _ = Describe("Testing compliance event formatting", Ordered, func() {
 		Expect(nonCompPlcEvents).To(BeEmpty())
 
 		By("Checking events on the parent policy")
-		Eventually(func() []v1.Event {
-			return utils.GetMatchingEvents(clientManaged, testNamespace,
-				case15AlwaysCompliantParentName, "policy: "+testNamespace+"/"+
-					case15AlwaysCompliantName, "^Compliant;", defaultTimeoutSeconds)
-		}, defaultTimeoutSeconds, 1).ShouldNot(BeEmpty())
+		Eventually(func(g Gomega) {
+			events := utils.GetMatchingEvents(
+				clientManaged,
+				testNamespace,
+				case15AlwaysCompliantParentName,
+				"policy: "+testNamespace+"/"+case15AlwaysCompliantName,
+				"^Compliant;",
+				defaultTimeoutSeconds,
+			)
+			g.Expect(events).ToNot(BeEmpty())
+
+			for _, event := range events {
+				g.Expect(event.Annotations[common.ParentDBIDAnnotation]).To(Equal("23"))
+				g.Expect(event.Annotations[common.PolicyDBIDAnnotation]).To(Equal("30"))
```

Comment:
Add a description here. If this fails it's going to be something like "" != "23", which isn't going to be clear.
```suggestion
				g.Expect(event.Annotations[common.ParentDBIDAnnotation]).To(Equal("23"), common.ParentDBIDAnnotation+" should have the correct  value")
				g.Expect(event.Annotations[common.PolicyDBIDAnnotation]).To(Equal("30"), common.PolicyDBIDAnnotation+" should have the correct  value")
```

Reply:
Oh I didn't know that we ca add description here

Reply:
Beautiful